src/state/apis/nft/selectors.ts

```diff
@@ -1,12 +1,14 @@
 import { QueryStatus } from '@reduxjs/toolkit/query'
 import type { AccountId, AssetId } from '@shapeshiftoss/caip'
+import { isNft } from '@shapeshiftoss/caip'
 import { createSelector } from 'reselect'
 import type { ReduxState } from 'state/reducer'
 import {
   selectEndpointNameParamFromFilter,
   selectQueryStatusParamFromFilter,
 } from 'state/selectors'
 import { selectWalletId } from 'state/slices/common-selectors'
+import { selectPortfolioAssetIds } from 'state/slices/selectors'
```

Comment:
Don't revert me just yet - when all is upserted, we should be able to select from portfolio data exclusively - i.e doing the `missingPortfolioNfts` logic here as a sanity check should now return an empty array

Reply:
Sanity check ✅ 
<img width="568" alt="image" src="https://github.com/shapeshift/web/assets/17035424/8b370ce0-11fc-4c9e-bb10-7e9c8e940dfe">